src/main/java/com/endava/pages/IndustriesPage.java

```diff
@@ -12,7 +12,8 @@
 
     private static final Logger logger = LoggerFactory.getLogger(IndustriesPage.class);
 
-	public By finance = By.xpath("//a[text()='Finance']");
+    //Find fix for xpath
```

Comment:
Did you find the fix or just forgot to delete the comment?

Reply:
Done.